scripts/validators.bash

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
```

Comment:
I really like the idea of separating this out! It was something I had thought about too but given I only needed one validation I didn't think it was worth splitting out 😄